buildroot/package/beocreate/beocreate.mk

```diff
@@ -48,6 +48,11 @@ define BEOCREATE_INSTALL_INIT_SYSTEMD
                 $(TARGET_DIR)/etc/systemd/system/beocreate2.service.d/override.conf
         $(INSTALL) -D -m 0644 $(BR2_EXTERNAL_HIFIBERRY_PATH)/package/beocreate/beocreate2.service \
                 $(TARGET_DIR)/lib/systemd/system/beocreate2.service
+
+	ifeq ($(VERSION),"0w")
```

Comment:
I'm not sure how to check for the device model or build config. I took this from https://github.com/hifiberry/hifiberry-os/blob/4fff8527510b66c2dfb1d0beb521f1f3e4da8c63/build-config#L81 but I fear this might not work here.

Guidance very much apprecianted!